v2/stacks/dataset-catalogue/Makefile

```diff
@@ -4,5 +4,8 @@ include ../common.mk
 .PHONY: init
 init: base-init
 	# Add stack specific initialisation logic here
+	gpg -d ../../../../dis-authentication-stub/static/keys/public.key.asc > ../../../../dis-authentication-stub/static/keys/public.key
```

Comment:
Might be better to use and environment variable here - see an example PATH_MANIFESTS